entity-types/ext-triton/triton_dashboard.json

```diff
@@ -0,0 +1,170 @@
+{
+  "name": "Triton",
+  "pages": [
+    {
+      "name": "Triton",
+      "description": null,
+      "widgets": [
+        {
+          "title": "Total CPU Memory (MB)",
```

Comment:
Have you tried the experience setting the unit in the dashboard and let the dashboard decide if showing in K,Mb,Gb ?